### ISSUE TITLE
VMs should autostart by default on XenServer

### ISSUE THREAD

**Issue by marcusbooyah** [nforgeio/neonKUBE#851]
On XCP-ng VMs will not autostart when XCP boots up. I think that they should by default, which is an option.

**Comment by jefflill** [comment]
Looks like we need to set this via the XCP CLI:
```
xe pool-param-set uuid=<pool-uuid> other-config:auto_poweron=true
```
https://github.com/xcp-ng/xcp/wiki/VM-Auto-Power-On

**Comment by jefflill** [comment]
Note that this auto start mode is **not compatible with HA enabled resource pools**.  We're going to detect this condition and will not enable autostart when HA is enabled.  We'll assume that the operator will manual configure the HA pool for this.

**Comment by jefflill** [comment]
Fixed for v2.2.0
